mono/utils/mono-threads-state-machine.c

```diff
@@ -18,49 +18,45 @@
 static int
 get_thread_state (int thread_state)
 {
-	MonoThreadStateMachine state;
-	state.raw = thread_state;
+	const MonoThreadStateMachine state = {thread_state};
```

Comment:
nit: this (and similar changes in the other helper functions) is an unrelated, unmotivated style change.

Reply:
A little shorter to read, but still clear.
A little more const.